Mathlib/Analysis/Meromorphic/Order.lean

```diff
@@ -240,6 +240,44 @@ theorem order_add_of_unequal_order (hf₁ : MeromorphicAt f₁ x) (hf₂ : Merom
   · simpa [h.le] using hf₁.order_add_of_order_lt_order hf₂ h
   · simpa [h.le, add_comm] using hf₂.order_add_of_order_lt_order hf₁ h
 
+/-- If a function is both meromorphic and continuous at a point, then it is analytic there. -/
+protected theorem analyticAt {f : 𝕜 → E} {x : 𝕜} (h : MeromorphicAt f x) (h' : ContinuousAt f x) :
+    AnalyticAt 𝕜 f x := by
+  cases ho : h.order with
+  | top =>
+    have : AnalyticAt 𝕜 (fun _ ↦ (0 : E)) x := analyticAt_const
+    apply this.congr
+    rw [← ContinuousAt.eventuallyEq_nhd_iff_eventuallyEq_nhdNE continuousAt_const h']
+    filter_upwards [h.order_eq_top_iff.1 ho] with y hy using by simp [hy]
+  | coe n =>
+    rcases h.order_eq_int_iff.1 ho with ⟨g, g_an, gx, hg⟩
+    have : 0 ≤ n := by
```

Comment:
Would a brief comment of the argument be helpful here? Or can it be usefully extracted into a separate lemma? (My intuition is yes: then doing so would be helpful.)

Reply:
(Aside: `MeromorphicAt.order_eq_nat_iff` does not exist, but a doc-string mentions this.)